app/models/concerns/relationship_bulk.rb

```diff
@@ -66,7 +69,7 @@ def run_update_callbacks(ids_json, extra_options_json)
       index_alias = CheckElasticSearchModel.get_index_alias
       es_body = []
       versions = []
-      callbacks = [:reset_counters, :update_counters, :propagate_inversion]
+      callbacks = [:turn_off_unmatched_field, :move_explainers_to_source, :apply_status_to_target]
```

Comment:
We didn't call these callbacks before? Or, are they here to replace the lines you deleted from this method?

Reply:
Yes, these callbacks replace the delete line, as I believe it's better to run callbacks for more consistency with other implementations.